gamemodes/cinema/gamemode/modules/theater/services/sv_hls.lua

```diff
@@ -1,18 +1,43 @@
-sv_GetVideoInfo = sv_GetVideoInfo or {}
-
-sv_GetVideoInfo.hls = function(self, key, ply, onSuccess, onFailure)
-	local onReceive = function(info)
-
-		http.Post( "https://swampservers.net/fedorabot/gis.php", {q=info.title}, 
-			function(body) 
-				info.thumb="http://swampservers.net/cinema/contain.php?i="..body 
-				onSuccess(info)
-			end, onFailure)
-	end
-
-	theater.GetVideoInfoClientside(self:GetClass(), key, ply, onReceive, onFailure)
-end
-
-
-
-			
+
+sv_GetVideoInfo = sv_GetVideoInfo or {}
+
+sv_GetVideoInfo.hls = function(self, key, ply, onSuccess, onFailure)
+
+	local match = string.match(key,"streamwat.ch/(%w+)/*$")
```

Comment:
prefer naming this variable like streamwatch_key or something (its totally not clear in other places that its related to streamwatch)